packages/di/src/common/services/InjectorService.ts

```
@@ -238,11 +237,11 @@
         return instance;
 
       case ProviderScope.REQUEST:
-        if (locals) {
-          locals.set(token, instance);
+        if (options.locals) {
+          options.locals.set(token, instance);
 
           if (provider.hooks && provider.hooks.$onDestroy) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)